app/src/main/java/com/xmartlabs/dd/ui/DashboardFragment.java

```diff
@@ -0,0 +1,111 @@
+package com.xmartlabs.dd.ui;
+
+import android.animation.ValueAnimator;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.github.florent37.diagonallayout.DiagonalLayout;
+import com.xmartlabs.dd.R;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+/**
+ * Created by scasas on 3/31/17.
+ */
+@SuppressWarnings("deprecation")
```

Comment:
? Is dangerous to apply this at a class level. You might be missing something if you do this.